logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/Conversation.kt

```diff
@@ -182,7 +183,7 @@ sealed class ConversationDetails(open val conversation: Conversation) {
             protocol = protocolInfo,
             mutedStatus = MutedConversationStatus.AllAllowed,
             removedBy = null,
-            lastNotificationDate = null,
+            lastNotificationDate = DateTimeUtil.currentIsoDateTimeString(),
```

Comment:
We shouldn't rely on local device's time, maybe for connection it should be `lastModifiedDate` or even just don't set a default value at all 🤔